docs/podman-events.1.md

```diff
@@ -0,0 +1,135 @@
+% podman-events(1)
+
+## NAME
+podman\-events- Monitor Podman events
+
+## SYNOPSIS
+**podman events** [*options*]
+
+## DESCRIPTION
+
+Monitor and print events that occur in Podman. Each event will include a timestamp,
+a type, a status, name (if applicable), and image (if applicable).
+
+Podman events will report the following event types:
```

Comment:
I think the man page should be rewritten for reporting events by category.

This looks like I could get a checkpoint event for a volume.